File: src/main/java/domain/Excursion.java
package domain;

import java.time.LocalTime;

public class Excursion extends Entity<Long>{
    private String company;
    private float price;
    private LocalTime time;
    private Long seats;

    public Excursion(String company, float price, LocalTime time, Long seats) {
        this.company = company;
        this.price = price;
        this.time = time;
        this.seats = seats;
    }

    public String getCompany() {
        return company;
    }

    public void setCompany(String company) {
        this.company = company;
    }

    public float getPrice() {
        return price;
    }

    public void setPrice(float price) {
        this.price = price;
    }

    public LocalTime getTime() {
        return time;
    }

    public void setTime(LocalTime time) {
        this.time = time;
    }

    public Long getSeats() {
        return seats;
    }

    public void setSeats(Long seats) {
        this.seats = seats;
    }
}

File: CSharp/CSharp/domain/Excursion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.domain
{
    public class Excursion : Entity<long>
    {
        public string Company { get; set; }
        public float Price { get; set; }
        public TimeSpan Time { get; set; }
        public long Seats { get; set; }

        public string Objective { get; set; }

        public Excursion(string company, float price, TimeSpan time, long seats, string objective)
        {
            Company = company;
            Price = price;
            Time = time;
            Seats = seats;
            Objective = objective;
        }
    }
}